cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -133,15 +133,16 @@ def _add_cfn_parameters(self):
     def _add_resources(self):
         # Add default build tags information
         tags = copy.deepcopy(self.config.build.tags) or []
-        tags.append(BaseTag(key="pcluster_build_image", value=utils.get_installed_version()))
+        tags.append(BaseTag(key=PCLUSTER_IMAGE_NAME_TAG, value=self.image_name))
         build_tags_map = {tag.key: tag.value for tag in tags}
         build_tags_list = [core.CfnTag(key=tag.key, value=tag.value) for tag in tags]
 
         # Add default ami tags information
         tags = copy.deepcopy(self.config.image.tags) if self.config.image and self.config.image.tags else []
-        tags.append(BaseTag(key="pcluster_version", value=utils.get_installed_version()))
+        tags.append(BaseTag(key="parallelcluster:version", value=utils.get_installed_version()))
```

Comment:
PCLUSTER_VERSION_TAG

Reply:
Then I think we should create a function to add all these tags, rather than having this duplicate code to appending all the tags.

Reply:
done